rlberry/agents/agent.py

```diff
@@ -74,6 +74,8 @@ class Agent(ABC):
     unique_id : str
         Unique identifier for the agent instance. Can be used, for example,
         to create files/directories for the agent to log data safely.
+    writer_extra : str in {"reward", "action", "action_and_reward"},
+        Scalar that will be recorded in the writer.
```

Comment:
I am thinking that this is not very accessible docstring because people will look into the docstring of stablebaselinesAgent and maybe AgentWithSimplePolicy but will not go till there. Maybe at some point we will want to copy important docstrings from Agent in both AgentWithSimplePolicy and in StableBaselinesAgent

Reply:
done !